identity-service/src/utils/withdrawUSDCInstructionsHelpers.js

```diff
@@ -0,0 +1,68 @@
+const {
+  ASSOCIATED_TOKEN_PROGRAM_ID,
+  TOKEN_PROGRAM_ID
+} = require('@solana/spl-token')
+const { Keypair } = require('@solana/web3.js')
+const config = require('../config')
+
+const solanaClaimableTokenProgramAddress = config.get(
+  'solanaClaimableTokenProgramAddress'
+)
+
+const allowedProgramIds = [
+  solanaClaimableTokenProgramAddress,
+  /* secp */ 'KeccakSecp256k11111111111111111111111111111'
+]
+
+const isRelayAllowedProgram = (instructions) => {
+  for (const instruction of instructions) {
+    if (!allowedProgramIds.includes(instruction.programId)) {
+      return false
+    }
+  }
+  return true
+}
+
+const checkCreateAccountInstruction = (instruction) => {
+  const isCreateInstruction =
+    instruction.programId === ASSOCIATED_TOKEN_PROGRAM_ID.toString()
+  return isCreateInstruction
+}
+
+const checkCloseAccountInstruction = (instruction) => {
+  const feePayerKeypairs = config.get('solanaFeePayerWallets')
+  const feePayerPubkeys = feePayerKeypairs.map((wallet) =>
+    Keypair.fromSecretKey(
+      Uint8Array.from(wallet.privateKey)
+    ).publicKey.toString()
+  )
+  const isCloseInstruction =
+    instruction.programId === TOKEN_PROGRAM_ID.toString() &&
+    instruction.data &&
+    instruction.data.data &&
+    instruction.data.data[0] === 9
```

Comment:
magic number, lets make a constant for `9`

Reply:
Actually we should probably use the enum here: https://github.com/solana-labs/solana-program-library/blob/fd479ffe6e016bf53cb4523e28ac88a0f83ccc3b/token/js/src/instructions/types.ts

Reply:
oo TY

Reply:
oh man it's in a later spl-token version :(

Reply:
didn't you just install spl-token into identity? also I'm guessing it's still there just located somewhere else or something?